docs/source/references/environment_variables.rst

```diff
@@ -27,6 +27,13 @@ Alice
     Password for Ethereum account used by Alice.
 
 
+Bob
+----
+
+* `NUCYPHER_BOB_ETH_PASSWORD`
+    Password for Ethereum account used by Bob.
```

Comment:
Does this PR change something wrt Bob's need of an ETH account? My understanding is that we don't require it at all.

Reply:
There was a slight change to character configuration tests, which includes bobs that support providers.  It's not *required* but we still want to check the cases where it is optionally used.